netx/net/sourceforge/jnlp/security/HttpsCertVerifier.java

```diff
@@ -184,10 +184,6 @@ private String getNamesForCert(X509Certificate c) {
                     }
                 }
             }
-
-            if (subjAltNames != null)
```

Comment:
I agree that the lines removed in this PR are incorrect.
But simply removing this neglects the case when `derValue.getAsString()` returns an empty string.